medusa/server/api/v2/series.py

```diff
@@ -96,18 +102,28 @@ def patch(self, series_slug, path_param=None):
         if indexer_id is not None and indexer_id != identifier.id:
             return self._bad_request('Conflicting series identifier')
 
-        done = {}
-        for key, value in data.items():
-            if key == 'pause':
-                if value is True:
-                    series.pause()
-                elif value is False:
-                    series.unpause()
-                else:
-                    return self._bad_request('Invalid request body: pause')
-                done[key] = value
-
-        return self._ok(done)
+        accepted = {}
+        ignored = {}
+        patches = {
```

Comment:
@ratoaq2 I know config uses self.patches but we need access to the series object which isn't available in the class scope.

Reply:
Ok if i'm understanding this correct, then what your doing here is getting the series object, and changing an attribute if it needs to be patched.

But the config uses this command, to store the new `in-mem` values into the db: `app.instance.save_config()`
You don't have something similar for the series object? Or i'm missing something. So these changes will not be persistent.

Reply:
The patches is a list of patchable fields, this is literally copied and pasted from the other apiv2 end points for patching.

Reply:
The only difference is config uses `self.patches` where as this uses a local variable called patches.